pytorch_lightning/accelerators/ddp_backend.py

```diff
@@ -104,11 +104,12 @@ def _call_children_scripts(self):
 
         os.environ['WORLD_SIZE'] = f'{num_gpus * self.trainer.num_nodes}'
 
-        self.trainer.interactive_ddp_procs = []
+        self.interactive_ddp_procs = []
         for local_rank in range(1, self.trainer.num_processes):
             env_copy = os.environ.copy()
             env_copy['LOCAL_RANK'] = f'{local_rank}'
             env_copy['PL_DDP_PID'] = str(self.trainer.data_parallel_device_ids[local_rank])
+            env_copy['PL_GLOBAL_SEED'] = os.environ.get('PL_GLOBAL_SEED', None)
```

Comment:
```suggestion
            env_copy['PL_GLOBAL_SEED'] = os.environ.get('PL_GLOBAL_SEED')
```
get('...', None) is redundant